src/devtools/client/webconsole/actions/messages.ts

```diff
@@ -42,7 +41,7 @@ export function setupMessages(store: UIStore) {
   ThreadFront.findConsoleMessages(
     (_, msg) => store.dispatch(onConsoleMessage(msg)),
     () => store.dispatch(onConsoleOverflow())
-  ).then(() => store.dispatch(messagesLoaded()));
+  ).then(() => store.dispatch(messagesLoaded()), Sentry.captureException);
```

Comment:
This si fine, but it would be better if we didn't have to sprinkle it in.

Perhaps we can add an exception higher up in the redux or protocol level